Horion/Module/Modules/Freecam.cpp

```diff
@@ -53,6 +55,6 @@ void Freecam::onDisable() {
 	if (g_Data.getLocalPlayer() != nullptr) {
 		g_Data.getLocalPlayer()->setPos(oldPos);
 		if (g_Data.getLocalPlayer()->gamemode != 1)
```

Comment:
i dont think theres need for "if (g_Data.getLocalPlayer()->gamemode != 1)" or the contents of it

Reply:
well it was in the original freecam so I didn't bother to take it out or tamper with it

Reply:
well theres no need for it as the module doesnt change the gamemode